setup.py

```diff
@@ -48,8 +48,8 @@
         'azure.storage.file',
     ],
     install_requires=[
-        'azure-nspkg==1.0.0',
-        'azure-common==0.20.0',
+        'azure-nspkg',
```

Comment:
Right.  azure-nspkg will never change.  azure-common might, but only in a backwards compatible way.  If we ever require something that isn't in all versions, we can set the dependency on azure-common to be minimum the version that has what we need.